portfolio/src/main/webapp/projects.html

```diff
@@ -0,0 +1,97 @@
+            The code helped me understand some Physics principles in
+            coding by simulating movement and rotation. You can
+            find the code on my Github account (link in the menu from the
+            right).<br><br>
+            This is how it turned out: <br><br>
+
+            <img src="/images/tetris.png" class="some-image">
+
+        </p>
+        <h4 class="title">School Simulation</h4>
+        <p>
+            This project was created to simulate a school: add courses,
+            teachers, certificates, etc. Courses have a number of free spaces
+            for students, as well as days in which they can be completed (be 
+            aware of weekends). <br>
+            Teachers have job specifications that permit them to teach a course.
+            Students can get certificates when they complete a course and can't
+            enroll in courses where they already have certificates. 
+            When a student has certificates for all the courses, he
+            finishes school. <br><br>
+            The aim is for all the students to finish all courses.
+            Programming language used: Java.
+        </p>
+        <h4 class="title">Mathdoku</h4>
+        <p>
+            Mathdoku, also known as KenKen, is a game that uses
+            math to be solved. For more details about the rules
+            of the game <a href="https://en.wikipedia.org/wiki/KenKen"
+            target="_blank">
+            click here</a>. It becomes more and more difficult depending
+            on the size of the board.
+            I created this game using Java and JavaFx to also interract
+            with the user.<br><br>
+            This is how this one turned out:<br>
+
+            <img src="/images/enabled.png" class="some-image">
+
+            <br> <br>
+
+            When the help option is enabled, if a row/column has 
+            numbers that repeat themselves, it highlights the
+            whole row/column with yellow; if a cage is filled with
+            numbers that don't respect the result, then it highlights
+            the cage with red. <br><br>
+            If the user wins, all the entries continue to color
+            themselves and it gives the impression of a disco ball.
+            The code for this one is not avilable because I commited
+            on the git from my university.
+        </p>
+        <h4 class="title">Coronavirus Data Analysis</h4>
+        <p>
+            This project was realised with SQL and bash script. The idea was
+            to analyse an official CSV file with data refering to current
+            coronavirus cases in each country, per day. Using SQL the
+            data was extracted and with bash script it was plotted.
+            <br><br>
+            This is how the plot looks:<br>
+
+            <img src="/images/graph.png" class="some-image">
+        </p>
+
+        <br><br><br>
+        </div>
+    </div>
+  </body>
+</html>
```

Comment:
minor comment: I would add a new line character at the end of every file, for consistency